### PR TITLE
Update SPARNA

### PR DESCRIPTION
**Pull request by Marie-Sparna**
adding pointers to SHACL Play

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,9 @@ Software tools or libraries, sorted by programming language.
...
 - [RDF4J SHACL Engine](https://github.com/eclipse-rdf4j/rdf4j/tree/main/core/sail/shacl) <img alt="Maven Central Version" src="https://img.shields.io/maven-central/v/org.eclipse.rdf4j/rdf4j-shacl" align="top"> <img alt="Maven Central Last Update" src="https://img.shields.io/maven-central/last-update/org.eclipse.rdf4j/rdf4j-shacl" align="top"> - Supports SHACL Core (without some property paths, see [5.0.3 improvements](https://github.com/Sveino/Inst4CIM-KG/issues/95#issuecomment-2437819932)), SHACL-SPARQL, incremental validation; [docs](https://rdf4j.org/documentation/programming/shacl/); `BSD-3-Clause` license; `Java`.
 - [TopBraid SHACL API](https://github.com/TopQuadrant/shacl) <img alt="Maven Central Version" src="https://img.shields.io/maven-central/v/org.topbraid/shacl" align="top"> <img alt="Maven Central Last Update" src="https://img.shields.io/maven-central/last-update/org.topbraid/shacl" align="top"> 💰 - Supports SHACL Core, SHACL-SPARQL, SHACL rules; based on Jena; `BSD-3-Clause` license; `Java`.
+  - [RDF validator](https://www.itb.ec.europa.eu/shacl/any/upload) - The EU Interoperability Test Bed (ITB) playground based on TQ API; validate any RDF with SHACL Shapes.
```
Comment (VladimirAlexiev) [inline README.md:37]: Your PR seems based on an older version of the README. There is duplication about ITB (lines 37, 38 and 40), please rebase it.

File: README.md [hunk 2]
```diff
@@ -34,6 +34,9 @@ Software tools or libraries, sorted by programming language.
...
+  - [SHACL shapes validator](https://www.itb.ec.europa.eu/shacl/shacl/upload) - The EU Interoperability Test Bed (ITB) playgrounds based on TQ API; validate SHACL Shapes itself.
+  - [Sparna SHACL validator](https://shacl-play.sparna.fr/play/validate) - Free online suite of tools from Sparna to work with SHACL; based on TQ API.
```
Comment (VladimirAlexiev) [inline README.md:39]: Let's not have 2 bullets for SPARNA (lines 39 and 41) in the same section.

**Review by VladimirAlexiev — CHANGES_REQUESTED** [review]

**Comment by Marie-Sparna** [comment]
Hi Vladimir,

Thank you for your message, I removed both additional ITB lines, can you
tell me if everything ok now ?

Best,

*Marie Muller* -* SPARNA*
Web de *données* | Architecture de l'*information* | Accès aux
*connaissances*
blog : blog.sparna.fr, site : sparna.fr


Le jeu. 12 déc. 2024 à 13:56, Vladimir Alexiev ***@***.***> a
écrit :

**Comment by VladimirAlexiev** [comment]
@Marie-Sparna Thanks a lot! Merged

**Pull request merged**